scripts/deploy.js

```diff
@@ -0,0 +1 @@
+const { SkynetClient } = require("");
```

Comment:
what is this used for?

Reply:
I think this wasn't meant to be committed lol. The deploy script is a separate PR